### PR TITLE
sha3 definition (fixes eslint error in master)

### PR DESCRIPTION
**Pull request by EdwardDrapkin**
Add a definition for the SHA3 library here: https://github.com/phusion/node-sha3

### CODE REVIEW

**Comment by EdwardDrapkin** [comment]
Build is gonna fail because eslint in master is broken

**Comment by EdwardDrapkin** [comment]
I dunno if it's a bug with eslint or what but eslint was erroring because another typedef had raw JSX in the same scope as a function declaration, so I fixed that by just wrapping the JSX in a function call.  Shouldn't affect that def's flow tests at all.

File: definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+declare module "sha3" {
+  declare class SHA3Hash {
+    constructor(hashLength?: 224 | 256 | 384 | 512): SHA3Hash;
+    update(data: any, inputEncoding?: 'binary' | 'ascii' | 'utf8'): void;
+    digest(type?: 'hex' | 'binary'): string;
+  }
+
+  declare module .exports: {
```
Comment (GAntoine) [inline definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js]: Remove the space in `module .exports`

**Comment by GAntoine** [comment]
> I dunno if it's a bug with eslint or what but eslint was erroring because another typedef had raw JSX in the same scope as a function declaration, so I fixed that by just wrapping the JSX in a function call. 

❤️

File: definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ declare module "sha3" {
     digest(type?: 'hex' | 'binary'): string;
   }
 
-  declare module .exports: {
+  declare module.exports: {
```
Comment (GAntoine) [inline definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js]: 👍

File: definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js (hunk 1 above)
Comment (EdwardDrapkin) [reply definitions/npm/sha3_v1.2.x/flow_v0.56.x-/sha3_v1.2.x.js]: fixed, I blame setting the save hotkey to 'format and save' in jetbrains >.>

**Pull request merged**